### PR TITLE
Change FB execution script for linux

### PR DESCRIPTION
**Pull request by mccarthyrb**
Change execution script from run-in-environ to flexbridge.
Keep run-in-environ for developer purposes.
Cleanup Makefile to include the flexbridge file.
Update Chorus*.tmx localization files to version 2.5.1068.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/126)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,15 +11,15 @@ all: release
...
 	./download_dependencies_linux.sh && . ./environ && cd build && xbuild FLExBridge.build.mono.proj /t:Build /p:RootDir=.. /p:teamcity_dotnet_nunitlauncher_msbuild_task=notthere /p:BUILD_NUMBER=$(BUILD_NUMBER) /p:BUILD_VCS_NUMBER=$(BUILD_VCS_NUMBER) /p:UploadFolder=$(UploadFolder) /p:Configuration=ReleaseMono /v:debug
-	cp -a run-in-environ environ environ-xulrunner output/ReleaseMono
+	cp -a flexbridge environ environ-xulrunner output/ReleaseMono
 
 debug: vcs_version
 	FBCommonAppData="/tmp/flexbridge"
 	if test ! -d "/tmp/flexbridge"; then mkdir -p "/tmp/flexbridge"; fi;
 	export FBCommonAppData
 	./download_dependencies_linux.sh && . ./environ && cd build && xbuild FLExBridge.build.mono.proj /t:Build /p:RootDir=.. /p:teamcity_dotnet_nunitlauncher_msbuild_task=notthere /p:BUILD_NUMBER=$(BUILD_NUMBER) /p:BUILD_VCS_NUMBER=$(BUILD_VCS_NUMBER) /p:UploadFolder=$(UploadFolder) /p:Configuration=DebugMono
 	# Put run-in-environment next to FLExBridge.exe, as it will be in a user's machine, so FW can easily find it on a developer's machine.
```
Comment (marksvc) [inline Makefile]: This comment will need updated to change `run-in-environment` to `flexbridge`

File: flexbridge [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Run command in environment
```
Comment (marksvc) [inline flexbridge]: How about instead:   # Launch FlexBridge

File: flexbridge [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+set -e -o pipefail
+
+scriptdir="$(dirname "$0")"
+prefix=$(cd "$scriptdir/../.."; /bin/pwd)
```
Comment (marksvc) [inline flexbridge]: I think that better than setting prefix this way and then using it on line 24 would be to test the beginning of the pwd string. But I don't think we need to make a change about that now.

File: flexbridge [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+cd "$(dirname "$0")"
+. environ
+
+(
```
Comment (marksvc) [inline flexbridge]: I'm a bit confused by the use of a subshell here. Why use a subshell in parentheses here? Ah, is it so XDG_DATA_HOME and FB_SHARE  don't stay set outside of the subshell?

File: flexbridge [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+		cp -a "/var/lib/flexbridge/localizations" "${FB_SHARE}"
+	else
+		# For developer build. This is still not working correctly.
+		cp -a "${prefix}/DistFiles/localizations" "${FB_SHARE}"
+	fi
+		
+)
+
+cd - >/dev/null
+"$@"
```
Comment (marksvc) [inline flexbridge]: Right, so I think line 34 will need to be set to something like

    "$(dirname "$0")/"FlexBridge.exe "$@"

**Comment by mccarthyrb** [comment]
Review status: 0 of 7 files reviewed at latest revision, 5 unresolved discussions.

---

*[flexbridge, line 8 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r1-8:-KUIfhM9rwPlXEBIkLOY:bef44sk) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/flexbridge#L8)):*
<details><summary><i>Previously, marksvc wrote…</i></summary>
> I think that better than setting prefix this way and then using it on line 24 would be to test the beginning of the pwd string. But I don't think we need to make a change about that now.
</details>
This is a work in progress for the developer build directory. If you can suggest better solutions I would be open to any.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126)*

**Comment by mccarthyrb** [comment]
Review status: 0 of 7 files reviewed at latest revision, 5 unresolved discussions.

---

*[flexbridge, line 3 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r1-3:-KUIp_GTaG7g0_k4iI1j:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/flexbridge#L3)):*
<details><summary><i>Previously, marksvc wrote…</i></summary>
> How about instead:   # Launch FlexBridge
</details>
Done.

---

*[flexbridge, line 8 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r1-8:-KUIpXsBmgTpsnXUzY1F:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/flexbridge#L8)):*
<details><summary><i>Previously, mccarthyrb (Brent McCarthy) wrote…</i></summary>
> This is a work in progress for the developer build directory. If you can suggest better solutions I would be open to any.
</details>
Done.

---

*[flexbridge, line 13 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r1-13:-KUIpHLB25nxNubsRLtt:bttiein) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/flexbridge#L13)):*
<details><summary><i>Previously, marksvc wrote…</i></summary>
> I'm a bit confused by the use of a subshell here. Why use a subshell in parentheses here? Ah, is it so XDG_DATA_HOME and FB_SHARE  don't stay set outside of the subshell?
</details>
Yes that is correct.

---

*[flexbridge, line 34 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r1-34:-KUIonmyIhBfh0UxuOoM:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/flexbridge#L34)):*
<details><summary><i>Previously, marksvc wrote…</i></summary>
> Right, so I think line 34 will need to be set to something like
> 
> 
> 
>     "$(dirname "$0")/"FlexBridge.exe "$@"
</details>
Done.

---

*[Makefile, line 21 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458A-r1-21:-KUIoitCXnh-VgO-kIDG:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/Makefile#L21)):*
<details><summary><i>Previously, marksvc wrote…</i></summary>
> This comment will need updated to change `run-in-environment` to `flexbridge`
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126)*

**Comment by marksvc** [comment]
:lgtm:

---

Review status: 0 of 7 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-:-KUIqBd6OvOYNZxYPtk4:bnfp4nl)*

**Comment by mccarthyrb** [comment]
Review status: 0 of 7 files reviewed at latest revision, 5 unresolved discussions.

---

*[Makefile, line 21 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458A-r1-21:-KUJ3Jb3FIOcU2LiHWV3:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/7dacca36c507ee3e82543740a7f7a7b839d3cca6/Makefile#L21)):*
<details><summary><i>Previously, mccarthyrb (Brent McCarthy) wrote…</i></summary>
> Done.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126)*

**Comment by mccarthyrb** [comment]
Review status: 0 of 7 files reviewed at latest revision, 4 unresolved discussions.

---

*[flexbridge, line 33 at r2](https://reviewable.io:443/reviews/sillsdev/flexbridge/126#-KUIZikptZvGs8ql458C-r2-33:-KUJ3ZcXFaq6mATD2D85:b-896fix) ([raw file](https://github.com/sillsdev/flexbridge/blob/9b04bd7bb6615701a217c9f6f85f03d3fbef3a42/flexbridge#L33)):*
<details><summary><i>Previously, papeh wrote…</i></summary>
> `FLExBridge.exe` (funny acronym case)
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126)*

**Comment by papeh** [comment]
Reviewed 5 of 7 files at r1, 1 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/126)*

**Pull request merged**